src/main/java/com/hpe/simulap/protocol/sip/sampler/SipSampler.java

```diff
@@ -1566,7 +1568,7 @@ private void receiveRequestMessage(SampleResult res) {
 
                 if ( aReq == null){
                     // No message received
-                	if (_logger.isDebugEnabled()) _logger.debug("incoming request for id "+id+ " not received within timeout "+timeout,null);
+			if (_logger.isDebugEnabled()) _logger.debug("incoming request for id "+id+ " not received within timeout "+timeout,"");
```

Comment:
debug method could be rewritten here like this `if (_logger.isDebugEnabled()) _logger.debug("incoming request for id {} not received within timeout {}, id, timeout);`